inspirehep/modules/workflows/tasks/arxiv.py

```diff
@@ -146,8 +146,13 @@ def arxiv_plot_extract(obj, eng):
                 return
 
             lb = LiteratureBuilder(source='arxiv', record=obj.data)
+            file_extension_pattern = re.compile('.+(\.[a-zA-Z]+)$')
```

Comment:
you don't need to reinvent the wheel, you can use [`os.path.splitext`](https://docs.python.org/2/library/os.path.html#os.path.splitext). In general, Python comes "batteries included", so you can assume the basic stuff is already out there. What you probably want to use though is [`os.path.basename`](https://docs.python.org/2/library/os.path.html#os.path.basename) instead of the `name` for the key.